Makefile

```diff
@@ -11,3 +13,4 @@ generate:
 get-deps:
 	go get -t -v ./...
 	go get -u golang.org/x/tools/cmd/stringer
+	go mod tidy
```

Comment:
this should supersede the need for the two `go get` in `get-deps` here

Reply:
good point. removed